data/lua/core/gui.lua

```diff
@@ -18,13 +25,10 @@ function gui.confirm(title, msg)
 	return gui.show_prompt(title, msg, "yes_no", true)
 end
 
---! Displays a WML message box with attributes from table @attr and options
---! from table @options.
---! @return the index of the selected option.
---! @code
---! local result = gui.get_user_choice({ speaker = "narrator" },
---!     { "Choice 1", "Choice 2" })
---! @endcode
+---Displays a WML message box with attributes from attr and options from options.
+---@param attr WML
+---@param options string[]
+---@return integer #the index of the selected option.
```

Comment:
Removing the example from the docs needs some documentation to replace it.

Reply:
How's c43b07e344452925570fd0f78e833ba5a0d0dff5 look?

Reply:
Ok